c/stack1.c

```diff
@@ -17,57 +17,54 @@ int count = 0;
 void main()
 {
 	int no,ch,e;
-       	printf("\n 1 - push");
-	printf("\n 2 - pop");
-        printf("\n 3 - top");
-        printf("\n 4 - empty");
-        printf("\n 5 - exit");
-    	printf("\n 6 - display");
-        printf("\n 7 - stack count");
-        printf("\n 8 - destroy");
-	create();
-	while(1){
-		printf("\n enter choice:");
-		scanf("%d",&ch);
+       		printf("\n 1 - push");
```

Comment:
use one tab space 
should be like 
int no,<space>ch, <space>e;
printf("");
printf("");
indent this way